README.md

```diff
@@ -4,4 +4,120 @@ The stilk of an apple is also called a pedicel.
 
 This Ruby gem will help you handle an Apple Pay `PKPaymentToken`.
 
+## Usage
 
+```ruby
+# Parse the payment token into a hash.
+apple_pay_payment_token_hash = JSON.parse(apple_pay_payment_token_json)
+
+# Instantiate.
+# Validates the format of the Apple Pay payment token; raises an
+# Pedicel::Validator::Error if the format is invalid.
+pedicel = Pedicel::EC.new(apple_pay_payment_token_hash)
```

Comment:
Minor comment: We could mention that the rules for the check exist at https://developer.apple.com/library/content/documentation/PassKit/Reference/PaymentTokenJSON/PaymentTokenJSON.html.

We should maybe mention that it does not support using RSA.

Reply:
What section do you refer to?

If the bullets below step 6 of the "to decrypt" steps: we cannot check that `currencyCode` and `transactionAmount` is correct (because that's not the job of this gem).

If the table below "Payment Data Keys": we do not ensure that it is indeed an ISO 4217 numeric code.

If we should refer to anything, then I think it should be the code. An addition could be a checklist of things the user of this gem should check on their own.

Reply:
You're right about step 6 :)

I leave it at your discretion, but I would refer to step 1, which is essentially what the checks boil down to.

Reply:
Ahh, I misunderstood what you meant (I thought about the `Validator` stuff, my bad).

"Validates the format" is not exactly right. It's more like "Validates the content". I'll adjust.

Reply:
The errors have "Format" in their name: https://github.com/clearhaus/pedicel/blob/2605cb741c10d9590392b80694a057fe9161751f/lib/pedicel/validator.rb#L11-L12

Should we adjust? To `TokenError` and `TokenDataError`? WDYT @kse-clearhaus @mt-clearhaus?

Reply:
The `Validator` stuff is more about format, so I guess it's acceptable. Otherwise, perhaps `TokenSchemaError` and `TokenDataSchemaError`?

Reply:
Hmm. It is about format, but not just, i.e. signature checking.
Maybe `TokenContentError` and `TokenDataContentError`.

But either will do.